docs-devsite/vertexai-preview.md

```diff
@@ -367,3 +371,27 @@ export declare enum HarmSeverity
 |  HARM\_SEVERITY\_NEGLIGIBLE | <code>&quot;HARM_SEVERITY_NEGLIGIBLE&quot;</code> |  |
 |  HARM\_SEVERITY\_UNSPECIFIED | <code>&quot;HARM_SEVERITY_UNSPECIFIED&quot;</code> |  |
 
+## VertexAIErrorCode
+
+Standardized error codes that [VertexAIError](./vertexai-preview.vertexaierror.md#vertexaierror_class) can have.
+
+<b>Signature:</b>
+
+```typescript
+export declare const enum VertexAIErrorCode 
+```
+
+## Enumeration Members
+
+|  Member | Value | Description |
+|  --- | --- | --- |
+|  ERROR | <code>&quot;error&quot;</code> | A generic error occured. |
```

Comment:
```suggestion
|  ERROR | <code>&quot;error&quot;</code> | A generic error occurred. |
```

Reply:
thanks for catching all these mistakes :p